### ISSUE TITLE
nvm node: Cannot start Tern Server

### ISSUE THREAD

**Issue by gennarocoppola8** [ternjs/tern#726]
I am on Mac os X, after installing tern and config it in the emacs.el file, i get nvm node: no such file or directory every time i should have completion, like when i digit . 
I installed nvm correctly and node through it. So i don't know what it oculd be the problem

**Comment by marijnh** [comment]
nvm seems to break executable hashbang-based JS scripts. Try manually setting your `tern-command` variable in emacs to whatever shell command (list of strings) that starts the `tern/bin/tern` script through nvm.

**Comment by gennarocoppola8** [comment]
I fixed everything. Things like these happened to me before and I would like to suggest to add this on the website for Mac Os X users. When Emacs looks for a program it goes in a list called exec-path, and not in your $PATH. So there is a package for emacs called exec-path-from-shell which has the aim to help Mac Os X users with emacs. But because to me this didn't work I added this line in my .emacs file:
(add-to-list 'exec-path "/usr/local/bin") that is the folder when npm install everything globally and brew too.
Doing this when he looks for nvm, tern or whatever it works. 
Another instersting thing is that when you run emacs in terminal mode(emacs -nw) this problem doesn't appear, when i understood it I made tern works like this. 
Great satisfation, I invite you to close this issue, but also to advice on the main page of the project this for emacs user under OS X.
:+1:
